resources/icons/power.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
all icons must be 24 x 24

Reply:
Removed all icons that were superseded by @yenda 's PR